packages/react-components/src/echarts/unboundedZoom/isDurationViewport.ts

```diff
@@ -0,0 +1,6 @@
+import { DurationViewport, Viewport } from '@iot-app-kit/core';
+
+export const isDurationViewport = (
```

Comment:
this also exists in the `components/chart` folder
noting it down so we can dedupe as a followup